components/speedreader/rust/ffi/BUILD.gn

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("config.gni")
+import("//brave/build/cargo.gni")
+import("//brave/test/testing.gni")
+
+config("external_config") {
+  include_dirs = [ "include" ]
+  defines = [
+    "NO_CXXEXCEPTIONS=1"
+  ]
+}
+
+source_set("speedreader_ffi") {
+  public_configs = [ ":external_config" ]
+
+  sources = [
+    "src/speedreader.cc",
+    "include/speedreader.h"
+  ]
+
+  deps = [
+    "//base",
+    ":speedreader"
+  ]
+
+  libs = [ speedreader_lib_path ]
+
+  if (is_mac || is_ios ) {
+    libs += [
+      "Security.framework"
```

Comment:
is this actually needed or was it just copied from another repo?

Reply:
copied, not noticed issues without, removed now